apache/superset
# refactor: Migrates legacy Sunburst charts to ECharts and removes legacy code

### SUMMARY
As part of the [4.0 approved initiatives](https://github.com/orgs/apache/projects/292/views/1), this PR removes the legacy Sunburst chart by:

- Removing old plugin references throughout the codebase
- Updating unit tests
- Updating E2E tests
- Updating the example dashboards

### TESTING INSTRUCTIONS
1 - Make sure all legacy Sunburst charts were converted to Sunburst v2
2 - Make sure legacy Sunburst is not available anymore in the viz picker

### ADDITIONAL INFORMATION
